hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/handler/UnassignRegionHandler.java

```diff
@@ -63,20 +63,25 @@ public class UnassignRegionHandler extends EventHandler {
 
   private boolean evictCache;
 
+  // active time of the master that sent this unassign request
+  private final long initiatingMasterActiveTime;
+
   public UnassignRegionHandler(HRegionServer server, String encodedName, long closeProcId,
     boolean abort, @Nullable ServerName destination, EventType eventType) {
-    this(server, encodedName, closeProcId, abort, destination, eventType, false);
+    this(server, encodedName, closeProcId, abort, destination, eventType, -1, false);
```

Comment:
I am not sure if anyone is using this. At least in the code, I didn't see any usage of this. As I was not aware I didn't change the signature. -1 is safe as it will not block the procedure reporting as -1 will be the smallest active code. 

Should I follow the aggressive approach and ask for an argument and pass that further? I think that would be good. Let me change to that approch

Reply:
I think best would be just remove it